clients/java/src/main/java/com/google/myanmartools/TransliterateU2Z.java

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.myanmartools;
+
+/**
+ * Transliteration initialization of phases and rules.
+ * Note: this is generated by compiling ICU transliteration rule format.
+ * Source rule file = genconvert/input/my-t-my-d0-zawgyi.txt
+ */
+
+import com.google.myanmartools.Transliterate;
+import com.google.myanmartools.Phase;
+import com.google.myanmartools.Rule;
+
+public class TransliterateU2Z extends Transliterate {
+  public TransliterateU2Z() {
+
```

Comment:
suggestion: call the parent constructor that takes a string name ID, not the default constructor

Reply:
Done